pmm-app/tests/verifyPMMSettingsPageElements_test.js

```diff
@@ -166,5 +145,48 @@ xScenario(
 
     await pmmSettingsPage.waitForPmmSettingsPageLoaded();
     I.waitForValue(pmmSettingsPage.fields.dataRetentionCount, dataRetention, 30);
-  }
+  },
+);
+
+Scenario(
+  'PMM-T415 - Verify Percona Platform (Sign up) elements on PMM Settings Page @not-pr-pipeline',
```

Comment:
These tests could be part of PR pipeline as well, as they don't need any data creation, please remove @not-pr-pipeline, in case they increase test run time, we can run them explicitly on FB's